OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -751,12 +743,12 @@ function Install-OctopusDeploy
     'admin',
     '--console',
     '--instance', $name,
-    '--username', $octopusAdminUsername,
```

Comment:
Not sure it's important, but it might have been easier to leave them with the same name - less change :)